PyInstaller/utils/win32/icon.py

```diff
@@ -169,8 +169,15 @@ def splitter(s):
     else:
         logger.info("Updating icons from %s to %s", srcpath, dstpath)
 
+    try:
+        hsrc = win32api.LoadLibraryEx(srcpath, 0, LOAD_LIBRARY_AS_DATAFILE)
+    except Exception as E :
```

Comment:
@tallforasmurf Can you take over this PR? This for sure going to fail the linter.